client/css/style.css

```diff
@@ -667,7 +667,8 @@ kbd {
 }
 
 #sidebar .lobby .add-channel.opened {
-	transform: rotate(45deg);
+	// translateZ(0) enables hardware acceleration, this is to avoid jittering when animiating
```

Comment:
How on Earth did that pass our CSS linter lol?!
Even my local linter (also stylelint) reported this, and actually whatever error I make in the CSS, `npm run lint:css` doesn't flag. There is something wrong in there, I'ma look right after...

Reply:
`animiating` :)

Reply:
Too used to writing SCSS :), fixing